api/src/__init__.py

```diff
@@ -0,0 +1,121 @@
+import requests
+from typing import List
+
+# ---------- GitHub API Classes ----------
+
+class GitHubIssue:
+    def __init__(self, issue_id, title, body, url, state):
+        self.id = issue_id
+        self.title = title
+        self.body = body
+        self.url = url
+        self.state = state
+
+class GitHubAPI:
```

Comment:
This class is irrelevant to the interface definition of an issue tracker client. It shouldn't be in this PR.